browser/src/control/Control.QuickFindPanel.ts

```diff
@@ -0,0 +1,140 @@
+// @ts-strict-ignore
+/* -*- js-indent-level: 8 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+/*
+ * JSDialog.QuickFindPanel
+ */
+
+/* global app */
+class QuickFindPanel {
+	map: any;
+	container: HTMLElement;
+	builder: any;
+
+	constructor(map: any) {
+		this.map = map;
+		this.container = document.querySelector('#quick-find-wrapper');
+	}
+
+	onAdd(map: any) {
+		// this.map = map;
+		this.map.on('quickfind', this.onQuickFind, this);
+		this.map.on('jsdialogupdate', this.onJSUpdate, this);
+		this.map.on('jsdialogaction', this.onJSAction, this);
+
+		this.builder = new L.control.jsDialogBuilder({
+			mobileWizard: this,
```

Comment:
nice to have:
windowId param missing:  should be new number which match the id in core: `desktop/source/lib/init.cxx:5141` 

see in notebookbar example or address input